### PR TITLE
remove unused import statement in keras dir

### PR DESCRIPTION
**Pull request by nzw0301**

### CODE REVIEW

File: keras/models.py [hunk 1]
```diff
@@ -5,7 +5,6 @@
 from . import backend as K
 from .engine.training import Model
 from .engine.topology import get_source_inputs, Node
-from .legacy.models import Graph
```
Comment (fchollet) [inline keras/models.py]: That's not an unused statement. Without it you can't import Graph from keras.models.

Comment (nzw0301) [inline keras/models.py]: thank you. fix my PR

**Comment by fchollet** [comment]
The build started failing after merging this. It's probably unrelated, but I'm reverting until we figure it out.

**Comment by fchollet** [comment]
Still failing. Almost certainly Travis being  broken.

**Pull request merged**